packages/go_router/pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: go_router
 description: A declarative router for Flutter based on Navigation 2 supporting
   deep linking, data-driven routes and more
-version: 13.0.1
+version: 13.0.2
 repository: https://github.com/flutter/packages/tree/main/packages/go_router
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+go_router%22
 
 environment:
   sdk: ">=3.0.0 <4.0.0"
-  flutter: ">=3.10.0"
+  flutter: ">=3.12.0"
```

Comment:
revert the flutter version change

Reply:
Should I revert the new changelog entry as well then?

Reply:
I believe we need to be on flutter >= 3.12.0 as [`RouteInformation.uri` does not exist on flutter 3.10.0](https://github.com/flutter/flutter/blob/84a1e904f44f9b0e9c4510138010edcc653163f8/packages/flutter/lib/src/widgets/router.dart#L41-L74).

See also: [here](https://github.com/flutter/flutter/issues/103659#issuecomment-1861808022)

Reply:
thanks for clarify this, looks like your test failed because the map here is not updated: https://github.com/flutter/packages/blob/f858453969324d13d4a98fc29a97f96d7c23df1b/script/tool/lib/src/common/core.dart#L61

Reply:
Seems Linux repo_checks is still failing after upping min flutter version to 3.13.0. I am unable to see what is causing this step to fail.

Reply:
the error messages are in checks-> linux repo_checks-> [View more details on flutter-dashboard] ->stdout 

https://logs.chromium.org/logs/flutter/buildbucket/cr-buildbucket/8758077883049129633/+/u/Run_package_tests/pubspec_validation/stdout

"  The minimum Dart version is 3.0.0, but the minimum Flutter version of 3.13.0 shipped with Dart 3.1.0. Please use consistent lower SDK bounds "

Reply:
Ah, thanks, I'll try to get that fixed up.

Reply:
Looks like everything's passing now, thanks @hangyujin